todo_app/app.py

```diff
@@ -1,15 +1,64 @@
-from flask import Flask
-
+import sys
+from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
+from todo_app.data.session_items import *
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+#error handling for 404
+@app.errorhandler(404)
+def not_found(e):
+    return render_template("404.html")
 
-@app.route('/')
+#default
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'
+    if request.method == 'POST':
+        task_title = request.form['title']
+        task_status = request.form['status']
+        add_item(title=task_title, status=task_status)
+        return redirect('/')
```

Comment:
This POST section is unused as you now have a separate path form for creating items.
The route can also be updated to be GET only.